pandas-stubs/core/frame.pyi

```diff
@@ -1295,6 +1295,7 @@ class DataFrame(NDFrame, OpsMixin):
         dropna: _bool = ...,
         margins_name: _str = ...,
         observed: _bool = ...,
+        sort: _bool = True,
```

Comment:
```suggestion
        sort: _bool = ...,
```
We don't put default args in the stubs unless they change the result of the method.